api/src/Microsoft.Azure.IIoT.OpcUa.Api/src/Core/Models/PendingAlarmApiModel.cs

```diff
@@ -0,0 +1,35 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.OpcUa.Api.Core.Models {
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Simple attribute operand model
+    /// </summary>
+    [DataContract]
+    public class PendingAlarmApiModel {
+
+        /// <summary>
+        /// Is pending alarm enabled?
+        /// </summary>
+        [DataMember(Name = "enabled", Order = 0)]
+        public bool Enabled { get; set; } = false;
```

Comment:
Actually, can we change this to IsEnabled?

Reply:
Sure, I'll fix.

Reply:
Done